src/main/java/com/google/maps/model/PlacesSearchResult.java

```diff
@@ -67,7 +67,7 @@
   /**
    * types contains an array of feature types describing the given result.
    */
-  public String types[];
+  public AddressType types[];
```

Comment:
It doesn't look like the code which sets up this value has changed? Or does the JSON decoder intelligently do the right thing for string enums here?